glusterd2/utils/pprof.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"net/http"
+	"net/http/pprof"
+	runtime_pprof "runtime/pprof"
+
+	"github.com/gorilla/mux"
+)
+
+var (
+	basePath string = "/debug/pprof/"
```

Comment:
no need to explicitly mention it as a string 
`basePath  = "/debug/pprof/"` should work

Reply:
done